Library/test-helpers/lib.sh

```diff
@@ -2032,6 +2280,11 @@ limeLibraryLoaded() {
         rlLogError "Failed loading library keylime/test-helpers."
         return 1
     fi
+
+    local PACKAGES="tpm2-tools openssl beakerlib podman"
+
+    echo -e "\nInstalled required packages for helping library if missing."
```

Comment:
weird wording, pls use: Install packages required by the library when missing.

Reply:
Sure, that sounds better.